Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
+            Admin.log("[core|add_filters] handle filter container: ", jQuery(e.target).attr("filter-container"));
+
+            const filters_container = jQuery(`#${jQuery(e.currentTarget).attr("filter-container")}`);
 
-            var filters_container = jQuery('#' + jQuery(e.currentTarget).attr('filter-container'));
+            if (jQuery("div[sonata-filter=\"true\"]:visible", filters_container).length == 0) {
 
-            if (jQuery('div[sonata-filter="true"]:visible', filters_container).length == 0) {
                 jQuery(filters_container).slideDown();
-            }
 
-            var targetSelector = jQuery(e.currentTarget).attr('filter-target'),
-                target = jQuery('div[id="' + targetSelector + '"]', filters_container),
-                filterToggler = jQuery('i', '.sonata-toggle-filter[filter-target="' + targetSelector + '"]')
-            ;
+            }
 
+            let targetSelector = jQuery(e.currentTarget).attr("filter-target"),
+                target = jQuery(`div[id="${targetSelector}"]`, filters_container),
+                filterToggler = jQuery("i", `.sonata-toggle-filter[filter-target="${targetSelector}"]`);
             if (jQuery(target).is(":visible")) {
-                filterToggler
-                    .removeClass('fa-check-square-o')
-                    .addClass('fa-square-o')
-                ;
 
+                filterToggler.
+                    removeClass("fa-check-square-o").
+                    addClass("fa-square-o");
                 target.hide();
 
             } else {
-                filterToggler
-                    .removeClass('fa-square-o')
-                    .addClass('fa-check-square-o')
-                ;
 
+                filterToggler.
+                    removeClass("fa-square-o").
+                    addClass("fa-check-square-o");
                 target.show();
+
             }
 
-            if (jQuery('div[sonata-filter="true"]:visible', filters_container).length > 0) {
+            if (jQuery("div[sonata-filter=\"true\"]:visible", filters_container).length > 0) {
+
                 jQuery(filters_container).slideDown();
+
             } else {
+
                 jQuery(filters_container).slideUp();
+
             }
+
         });
 
-        jQuery('.sonata-filter-form', subject).on('submit', function () {
-            jQuery(this).find('[sonata-filter="true"]:hidden :input').val('');
+        jQuery(".sonata-filter-form", subject).on("submit", function () {
+
+            jQuery(this).find("[sonata-filter=\"true\"]:hidden :input").
+                val("");
+
         });
 
         /* Advanced filters */
-        if (jQuery('.advanced-filter :input:visible', subject).filter(function () { return jQuery(this).val() }).length === 0) {
-            jQuery('.advanced-filter').hide();
-        };
+        if (jQuery(".advanced-filter :input:visible", subject).filter(function () {
```

Comment:
- eslint: Line 354 exceeds the maximum line length of 80.
- eslint: Unexpected unnamed function.